src/utils/csrf-token.ts

```diff
@@ -0,0 +1,7 @@
+export function getCSRFToken(): string | null {
+	return localStorage.getItem('CSRFToken');
```

Comment:
This is a little different from legacy since we don't use an HTTPOnly cookie, but I think it should still be OK.

The only attack I can think of would be a Zimlet stealing the CSRF token in order to forge requests, but at that point the Zimlet already has the power to make API requests with the users real auth token so it doesn't matter.